lib/Worker.js

```diff
@@ -21,23 +21,24 @@ class Worker extends EventEmitter {
   /**
    * @param {Puppeteer.CDPSession} client
    * @param {string} url
-   * @param {function(!Protocol.Log.entryAddedPayload)} logEntryAdded
+   * @param {function(!string, !Array<!JSHandle>)} consoleAPICalled
```

Comment:
`!Puppeteer.JSHandle`?

Reply:
We haven't been using the Puppeteer. namespace when the type is included directly in the file.

Reply:
Ack